stripe/src/main/java/com/stripe/android/paymentsheet/ui/BasePaymentSheetActivity.kt

```diff
@@ -90,7 +89,7 @@ internal abstract class BasePaymentSheetActivity<ResultType> : AppCompatActivity
         result: ResultType
     ) {
         setActivityResult(result)
-        hideSheet()
+        bottomSheetController.hide()
```

Comment:
Let's rename `BasePaymentSheetActivity#animateOut()` to something like `close()`?